### ISSUE TITLE
Inventory-next set up in staging

### ISSUE THREAD

**Issue by chris-macdermaid** [GSA/data.gov#2147]
## User Story
Now that inventory-next is in the staging playbook and deploying to staging has been tested, complete the following set up steps to get inventory-next running irrespective of migration issues yet to be resolved.

## Details/tasks
- [x] New Load Balancer and/or Netscaler configuration in BSP (ticket to BSP)
- [x] Request DNS (ticket to BSP)
- [x] Run db init commands
- [x] Create new db for datapusher - using a modified version of the db_init.py script

## Acceptance Criteria
- [x] GIVEN inventory-next has been deployed to staging\
  AND set up has been competed\
  WHEN I browse https://inventory-next-datagov.dev-ocsit.bsp.gsa.gov\
  THEN I see the inventory-next home page\
  OR browse request causes CKAN error and new ticket is created to resolve.

## Background
The playbook for staging was completed in https://github.com/gsa/datagov-deploy/issues/2118
The SSL cert has already been added https://github.com/GSA/datagov-deploy/blob/develop/ansible/bin/staging-sans.txt#L8

## Security Considerations ([required](https://nvd.nist.gov/800-53/Rev4/control/CM-4))
[comment]: # "Our SSP says 'The Data.gov team ensures security implications are considered as part of the agile requirements refinement process by including a section in the issue template used as a basis for new work.' so please don't remove this section without care."
The deployment will use the usual processes for staging/production rollouts, and not introduce any new process or components where security problems are likely to appear.

## Sketch
Prerequisites: Database, Redis, Solr, etc should all exist already and credentials have been added to the vault.
